### PR TITLE
Implement View For Selecting Post Tags

### PR DESCRIPTION
**Pull request by OnyekachiSamuel**
**What this PR does**:
- This PR handles the implementation of the view for selecting tags when creating a post
### Checklist

- [✅] Compiler warnings resolved
- [✅] Linter warnings resolved
- [✅] User-facing strings in Localizable.strings file
- [✅] Unused code, print statements, and comments removed

**Which issue(s) this PR fixes**:
#61

Resolves #

**Special notes for reviewers**:
Screenshot 2020-05-18 at 08 54 58

### CODE REVIEW

**Review by ethanswift — APPROVED** [review]
nice work!

**Review by ethanswift — APPROVED** [review]
Nice work! the add tag button doesn't have border in the image, please fix this before merging,

File: FightPandemics/Features/Post/CreatePost/PostCreationTagView.swift [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+        } else if tagCount == 1 {
+            return "Tag-Btn-With-One-Selected-Tag".localized
+        } else {
+            return "Tag-Btn-With-Many-Selected-Tags".localized
+        }
+    }
+
+    private lazy var addTagButton: UIButton = {
+        let button = UIButton()
+        let localizedString = String.localizedStringWithFormat(stringFormat, 0)
```
Comment (haidalinda) [inline FightPandemics/Features/Post/CreatePost/PostCreationTagView.swift]: consider using the `func localized(formattedWith:)` utility in `String+FightPandemics`. not required! just an attempt to make something like this less verbose

**Review by haidalinda — APPROVED** [review]
very nice !

File: FightPandemics/Features/Post/CreatePost/PostCreationTagView.swift (hunk 1 above)
Comment (OnyekachiSamuel) [reply FightPandemics/Features/Post/CreatePost/PostCreationTagView.swift]: Okay, thanks 👍🏻

**Review by stavares843 — COMMENTED** [review]
when tapping on add a tag doesn't trigger that view 

https://jmp.sh/ACNwz8Q

is that for a separate PR?

**Pull request merged**